README.md

```diff
@@ -1,80 +1,39 @@
 # Algoritmekader
+We ontwikkelen het Algoritmekader op een open manier via Github. Bekijk [de ontwikkelomgeving van het Algoritmekader](https://minbzk.github.io/Algoritmekader/). 
 
-**Welkom op de Github repo voor het algoritmekader!**
+**Dit is de Github repo voor het Algoritmekader!**
+Op deze pagina lees je hoe je kunt bijdragen aan de ontwikkeling van het Algoritmekader. Boven deze tekst vind je de bestanden voor het kader. We gebruiken Markdown en Mkdocs. Meer daarover lees je onder punt 5. 
 
-## Algoritmekader
+## Hoe kun je bijdragen?
+Dat kan op verschillende manieren. Kijk welke manier jou het beste past. 
+1 t/m 5 zijn geschikte manieren voor wie die nog niet zo bekend zijn met Github. 
```

Comment:
Voor "wie die" zou ik aanpassen naar voor "diegene die". Of de zin aanpassen naar. Ben je nog niet zo bekend met Github? Check dan eerst de stappen 1 t/m 5.